lib/iiif_manifest/v3/manifest_builder/record_property_builder.rb

```diff
@@ -54,8 +56,9 @@ def setup_manifest_from_record(manifest, record)
           manifest.rendering = populate_rendering if populate_rendering.present?
           homepage = ::IIIFManifest.config.manifest_value_for(record, property: :homepage)
           manifest.homepage = homepage if homepage.present?
+          apply_thumbnail_to(manifest)
```

Comment:
Could this be taken from the child canvas instead of being computed again?  Maybe something like `manifest.items.first&.thumbnail` would be sufficient?  Although it might not be populated until after `canvas_builder.apply(manifest.items)` on line 21.

Reply:
I wish it was that easy haha, here's a [PR](https://github.com/samvera/iiif_manifest/pull/80) that I think would achieve this.  My approach was when the canvas `thumbnail` gets set, I set the manifest `thumbnail` as well, if it doesn't exist already.  To achieve that, I had to pass manifest in and that affected a lot more files than originally anticipated.

Reply:
Another idea i had was would it be better to make the thumbnail on the manifest configurable?

Reply:
Yes, that's a great idea.  I think like with canvases `iiif_manifest` should allow configuring the thumbnail and can fall back to using the first canvas's thumbnail if not already set.

Reply:
That sounds good, can you merge in your branch and I'll take care of linting and then I can work on the configuration?

Reply:
I merged it in and resolved the rubocop issues.